### ISSUE TITLE
[BUG] crashed when instalizing the game

### ISSUE THREAD

**Issue by Braydy** [PojavLauncherTeam/PojavLauncher#2215]
### Describe the bug

Game exited

### The log file and images/videos

![Screenshot_20211108_102229](https://user-images.githubusercontent.com/93697405/140679409-c549bf79-8219-4027-94b6-085d9c421294.jpg)
![Screenshot_20211108_102233](https://user-images.githubusercontent.com/93697405/140679423-b98354d9-7c9c-4c49-bc7e-da8f3f37908d.jpg)

### Steps To Reproduce

```markdown
1. Start pojavlauncher
2. Launch labymod client.
3. instalizing
4. crashes (game exited)
```

### Expected Behavior

I expected it launched but it crashes

### Platform

```markdown
- Device model: vivo 1904
- CPU architecture: 2/3 Ram
- Android version: 11
- PojavLauncher version: lastest version 3.3.1.1
```

### Anything else?

Game exited

**Comment by Braydy** [comment]
How can I fix this

**Comment by ajit850** [comment]
Please send latestlog.txt

**Comment by Braydy** [comment]
Wait

**Comment by Braydy** [comment]
How do I send`

**Comment by Braydy** [comment]
I'm not sure but it's says this

**Comment by Braydy** [comment]
--------- beggining with launcher debug
Info: Launcher version: crocus-v3_openjdk
Info: LWJGL3 directory: [version, lwjgl-stb.jar, lwjgl-opengl.jar, lwjgl-tinyfd.jar, lwjgl-glfw-classes.jar, lwjgl.jar, lwjgl-jemalloc.jar, jsr305.jar, lwjgl-openal.jar]
Architecture: arm64
Info: Custom Java arguments: "-Xms550m -Xmx550m"
Info: Selected Minecraft version: LabyMod-3-1.8.9
Added custom env: TMPDIR=/data/user/0/net.kdt.pojavlaunch/cache
Added custom env: AWTSTUB_WIDTH=1544
Added custom env: REGAL_GL_VERSION=4.5
Added custom env: REGAL_GL_VENDOR=Android
Added custom env: LIBGL_MIPMAP=3
Added custom env: allow_higher_compat_version=true
Added custom env: MESA_GLSL_CACHE_DIR=/data/user/0/net.kdt.pojavlaunch/cache
Added custom env: HOME=/storage/emulated/0/Android/data/net.kdt.pojavlaunch/files/.minecraft
Added custom env: PATH=/data/user/0/net.kdt.pojavlaunch/runtimes/Internal/bin:/product/bin:/apex/com.android.runtime/bin:/apex/com.android.art/bin:/system_ext/bin:/system/bin:/system/xbin:/odm/bin:/vendor/bin:/vendor/xbin
Added custom env: force_glsl_extensions_warn=true
Added custom env: LIBGL_NORMALIZE=1
Added custom env: LD_LIBRARY_PATH=/data/user/0/net.kdt.pojavlaunch/runtimes/Internal/lib/aarch64/jli:/data/user/0/net.kdt.pojavlaunch/runtimes/Internal/lib/aarch64:/system/lib64:/vendor/lib64:/vendor/lib64/hw:/data/app/~~-SFWuRPo_9KH_5GYzVTRjg==/net.kdt.pojavlaunch-Wqv0JH6vCP4H0XefywSZsw==/lib/arm64
Added custom env: POJAV_RENDERER=opengles2_5
Added custom env: LIBGL_ES=2
Added custom env: MESA_LOADER_DRIVER_OVERRIDE=zink
Added custom env: MESA_GLSL_VERSION_OVERRIDE=460
Added custom env: JAVA_HOME=/data/user/0/net.kdt.pojavlaunch/runtimes/Internal
Added custom env: MESA_GL_VERSION_OVERRIDE=4.6
Added custom env: allow_glsl_extension_directive_midshader=true
Added custom env: REGAL_GL_RENDERER=Regal
Added custom env: AWTSTUB_HEIGHT=720
Initialising gl4es
v1.1.5 built on Aug 18 2021 16:47:47
Using GLES 2.0 backend
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
loaded: libGLESv2.so
loaded: libEGL.so
Using GLES 2.0 backend
Hardware Full NPOT detected and used
Extension GL_EXT_blend_minmax  detected and used
Extension GL_EXT_draw_buffers  detected and used
FBO are in core, and so used
PointSprite are in core, and so used
CubeMap are in core, and so used
BlendColor is in core, and so used
Blend Subtract is in core, and so used
Blend Function and Equation Separation is in core, and so used
Texture Mirrored Repeat is in core, and so used
Extension GL_OES_mapbuffer  detected
Extension GL_OES_element_index_uint  detected and used
Extension GL_OES_packed_depth_stencil  detected and used
Extension GL_OES_depth24  detected and used
Extension GL_OES_rgb8_rgba8  detected and used
Extension GL_EXT_multi_draw_arrays  detected
Extension GL_EXT_texture_format_BGRA8888  detected and used
Extension GL_OES_depth_texture  detected and used
Extension GL_OES_texture_stencil8  detected and used
Extension GL_EXT_texture_rg  detected and used
Extension GL_OES_texture_float  detected and used
Extension GL_OES_texture_half_float  detected and used
Extension GL_EXT_color_buffer_float  detected and used
Extension GL_EXT_color_buffer_half_float  detected and used
Extension GL_OES_fragment_precision_high  detected and used
Extension GL_EXT_shader_texture_lod detected and used
Max vertex attrib: 16
Extension GL_OES_standard_derivatives  detected and used
Max texture size: 8192
Max Varying Vector: 15
Texture Units: 16/16 (hardware: 16), Max lights: 8, Max planes: 6
Max Color Attachments: 8 / Draw buffers: 8
Hardware vendor is Imagination Technologies
GLSL 300 es supported
GLSL 310 es supported and used
sRGB surface supported
EGLImage to Texture2D supported
EGLImage to RenderBuffer supported
--------- beginning of crash
ignore MipMap
Targeting OpenGL 2.1
NPOT texture handled in hardware
Not trying to batch small subsequent glDrawXXXX
try to use VBO
FBO workaround for using binded texture enabled
Force texture for Attachment color0 on FBO
Hack to trigger a SwapBuffers when a Full Framebuffer Blit on default FBO is done
Force normals to be normalized on FPE shaders
glX Will try to recycle EGL Surface
Current folder is:/
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
--------- beginning of crash
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
--------- beginning of crash
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
--------- beginning of crash
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
--------- beginning of crash
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
OpenJDK 64-Bit Server VM warning: No monotonic clock was available - timed services may be adversely affected if the time-of-day clock changes

--------- beginning of crash
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
--------- beginning of crash
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
--------- beginning of crash
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
--------- beginning of crash
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
--------- beginning of crash
logcat: Unexpected EOF!

This means that either the device shut down, logd crashed, or this instance of logcat was unable to read log
messages as quickly as they were being produced.

If you have enabled significant logging, look into using the -G option to increase log buffer sizes.
ERROR: Unable to get more log.
[04:51:44] [main/INFO]: Loading tweak class name net.labymod.vanillaforge.LabyModTweaker
[04:51:44] [main/INFO]: Using primary tweak class name net.labymod.vanillaforge.LabyModTweaker
[04:51:44] [main/INFO]: Calling tweak class net.labymod.vanillaforge.LabyModTweaker
[LabyMod-Vanilla] Detected version: 1.8

[04:51:48] [main/INFO]: Registering transformer net.labymod.core.asm.LabyModTransformer
[04:51:48] [main/INFO]: Registering transformer net.labymod.addons.itemphysic.asm.ItemPhysicTransformer
[04:51:48] [main/INFO]: Registering transformer net.labymod.addons.moreparticle.asm.MoreParticleTranformer
[04:51:48] [main/INFO]: Registering transformer net.labymod.addons.resourcepacks24.asm.Resourcepacks24Transformer
[04:51:48] [main/INFO]: Registering transformer net.labymod.addons.custommainmenu.asm.CustomMainMenuTransformer
[04:51:48] [main/INFO]: Registering transformer net.labymod.addons.motionblur.asm.MotionBlurTransformer
[04:51:48] [main/INFO]: Launching wrapped minecraft {net.minecraft.client.main.Main}
[04:51:48] [main/INFO]: [LabyMod-Vanilla] Transforming class axu
[04:51:49] [main/INFO]: [LabyMod-Vanilla] Transforming class ave
[04:51:49] [main/INFO]: [LabyMod-Vanilla] Transforming class bet
[04:51:50] [main/INFO]: [LabyMod-Vanilla] Transforming class net.minecraftforge.client.event.GuiOpenEvent
[04:51:50] [main/INFO]: [LWJGL] Failed to load a library. Possible solutions:
	a) Add the directory that contains the shared library to -Djava.library.path or -Dorg.lwjgl.librarypath.
	b) Add the JAR that contains the shared library to the classpath.
[04:51:50] [main/INFO]: [LWJGL] Enable debug mode with -Dorg.lwjgl.util.Debug=true for better diagnostics.
[04:51:50] [main/INFO]: [LWJGL] Enable the SharedLibraryLoader debug mode with -Dorg.lwjgl.util.DebugLoader=true for better diagnostics.
[04:51:58] [Client thread/INFO]: [LabyMod-Vanilla] Transforming class net.minecraftforge.client.event.RenderGameOverlayEvent
EGLBridge: Initializing
EGLBridge: Initialized!
EGLBridge: ThreadID=19367
EGLBridge: EGLDisplay=0x1, EGLSurface=0x79fb47b080
EGLBridge: Created CTX pointer = 0x79fb47bb80
[04:51:58] [Client thread/INFO]: LWJGLX: ready-handle context created
[04:51:58] [Client thread/ERROR]: catching
java.lang.Exception: Trace exception
	at org.lwjgl.glfw.GLFW.glfwMakeContextCurrent(GLFW.java:1004) [lwjgl-glfw-classes.jar:?]
	at org.lwjgl.opengl.ContextGL.makeCurrent(ContextGL.java:197) [lwjgl-glfw-classes.jar:?]
	at org.lwjgl.opengl.Display.create(Display.java:419) [lwjgl-glfw-classes.jar:?]
	at org.lwjgl.opengl.Display.<clinit>(Display.java:91) [lwjgl-glfw-classes.jar:?]
	at Config.getDisplayModes(Config.java:1862) [Config.class:?]
	at avh$a.<clinit>(GameSettings.java:3384) [avh$a.class:?]
	at avh.<init>(GameSettings.java:325) [avh.class:?]
	at ave.am(SourceFile:349) [ave.class:?]
	at ave.a(SourceFile:310) [ave.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:124) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0-internal]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0-internal]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0-internal]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0-internal]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
EGLBridge: Comparing: thr=19367, this=0x79fb47bb80, curr=0x0
EGLBridge: Making current on window 0x79fb47bb80 on thread 19367
EGLBridge: eglMakeCurrent() succeed!
[04:51:58] [Client thread/INFO]: 79fb47bb80
[04:51:58] [Client thread/INFO]: LWJGLX: GL caps init
EGLBridge: Comparing: thr=19367, this=0x79fb47bb80, curr=0x79fb47bb80
[04:51:59] [Client thread/INFO]: Grab: false
[04:51:59] [Client thread/INFO]: [OptiFine] *** Reflector Forge ***
[04:52:00] [Client thread/INFO]: [LabyMod-Vanilla] Transforming class bfk
[04:52:00] [Client thread/INFO]: [LabyMod-Vanilla] Transforming class biu
[04:52:00] [Client thread/INFO]: [OptiFine] *** Reflector Vanilla ***
[04:52:00] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.fml.common.eventhandler.Event$Result
[04:52:00] [Client thread/INFO]: LWJGL Version: 3.2.3 SNAPSHOT
[04:52:00] [Client thread/ERROR]: catching
java.lang.NullPointerException
	at org.lwjgl.system.Pointer$Default.<init>(Pointer.java:88) ~[lwjgl.jar:SNAPSHOT]
	at org.lwjgl.system.CustomBuffer.<init>(CustomBuffer.java:25) ~[lwjgl.jar:SNAPSHOT]
	at org.lwjgl.system.StructBuffer.<init>(StructBuffer.java:20) ~[lwjgl.jar:SNAPSHOT]
	at org.lwjgl.glfw.GLFWImage$Buffer.<init>(GLFWImage.java:325) ~[lwjgl-glfw-classes.jar:?]
	at org.lwjgl.opengl.Display.setIcon(Display.java:951) [lwjgl-glfw-classes.jar:?]
	at ave.ar(SourceFile:560) [ave.class:?]
	at ave.am(SourceFile:361) [ave.class:?]
	at ave.a(SourceFile:310) [ave.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:124) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0-internal]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0-internal]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0-internal]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0-internal]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
[04:52:00] [Client thread/INFO]: LWJGLX: switch fullscreen to true
[04:52:00] [Client thread/INFO]: LWJGLX: switch fullscreen to true
EGLBridge: Created CTX pointer = 0x79fb419780
[04:52:00] [Client thread/INFO]: LWJGLX: ready-handle context created
[04:52:00] [Client thread/ERROR]: catching
java.lang.Exception: Trace exception
	at org.lwjgl.glfw.GLFW.glfwMakeContextCurrent(GLFW.java:1004) [lwjgl-glfw-classes.jar:?]
	at org.lwjgl.opengl.ContextGL.makeCurrent(ContextGL.java:197) [lwjgl-glfw-classes.jar:?]
	at org.lwjgl.opengl.Display.create(Display.java:419) [lwjgl-glfw-classes.jar:?]
	at org.lwjgl.opengl.Display.create(Display.java:253) [lwjgl-glfw-classes.jar:?]
	at ave.ap(SourceFile:521) [ave.class:?]
	at ave.am(SourceFile:363) [ave.class:?]
	at ave.a(SourceFile:310) [ave.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:124) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0-internal]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0-internal]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0-internal]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0-internal]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
EGLBridge: Comparing: thr=19367, this=0x79fb419780, curr=0x79fb47bb80
[04:52:00] [Client thread/INFO]: 79fb47bb80
[04:52:00] [Client thread/INFO]: [OptiFine] 
[04:52:00] [Client thread/INFO]: [OptiFine] OptiFine_1.8.9_HD_U_M5
[04:52:00] [Client thread/INFO]: [OptiFine] Build: 20210124-163719
[04:52:00] [Client thread/INFO]: [OptiFine] OS: Linux (aarch64) version Android-11
[04:52:00] [Client thread/INFO]: [OptiFine] Java: 1.8.0-internal, Oracle Corporation
[04:52:00] [Client thread/INFO]: [OptiFine] VM: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
[04:52:00] [Client thread/INFO]: [OptiFine] LWJGL: 3.2.3 SNAPSHOT
[04:52:00] [Client thread/INFO]: [OptiFine] OpenGL: GL4ES wrapper, version 2.1 gl4es wrapper 1.1.5, ptitSeb
[04:52:00] [Client thread/INFO]: [OptiFine] OpenGL Version: 2.1.0
[04:52:00] [Client thread/INFO]: [OptiFine] OpenGL Occlussion culling: Not available (GL_ARB_occlusion_query)
[04:52:00] [Client thread/INFO]: [OptiFine] Maximum texture size: 8192x8192
[04:52:00] [VersionCheck/INFO]: [OptiFine] Checking for new version
[04:52:00] [Client thread/INFO]: [Shaders] OpenGL Version: 2.1 gl4es wrapper 1.1.5
[04:52:00] [Client thread/INFO]: [Shaders] Vendor:  ptitSeb
[04:52:00] [Client thread/INFO]: [Shaders] Renderer: GL4ES wrapper
[04:52:00] [Client thread/INFO]: [Shaders] Capabilities:  2.0  2.1  -  -  - 
[04:52:00] [Client thread/INFO]: [Shaders] GL_MAX_DRAW_BUFFERS: 8
[04:52:00] [Client thread/INFO]: [Shaders] GL_MAX_COLOR_ATTACHMENTS_EXT: 8
[04:52:00] [Client thread/INFO]: [Shaders] GL_MAX_TEXTURE_IMAGE_UNITS: 16
[04:52:00] [Client thread/INFO]: [Shaders] Load shaders configuration.
[04:52:00] [Client thread/INFO]: [Shaders] Shaders can not be loaded, Fast Render is enabled.
[04:52:00] [Client thread/INFO]: [Shaders] No shaderpack loaded.
[04:52:00] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.fml.common.Loader
[04:52:00] [Client thread/INFO]: [OptiFine] VboRegions not supported, missing: OpenGL 1.3, ARB_copy_buffer
[04:52:01] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.fml.client.SplashProgress
[04:52:01] [Client thread/INFO]: Reloading ResourceManager: Default, !        Â§4Magma 32x Â§bV2.zip
[04:52:01] [VersionCheck/INFO]: [OptiFine] Version found: M5
[04:52:01] [Client thread/INFO]: [OptiFine] *** Reloading textures ***
[04:52:01] [Client thread/INFO]: [OptiFine] Resource packs: !        Â§4Magma 32x Â§bV2.zip
[04:52:01] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.client.model.Attributes
[04:52:01] [Client thread/INFO]: [OptiFine] (Reflector) Method not present: bmv$b.preDraw
[04:52:01] [Client thread/INFO]: [OptiFine] (Reflector) Method not present: bmv$b.postDraw
[04:52:01] [Client thread/INFO]: [OptiFine] Loading mcpatcher/font/ascii.properties
[04:52:02] [Client thread/INFO]: [LabyMod-Vanilla] Transforming class avo
[04:52:02] [Client thread/INFO]: If on Windows, make sure to provide all of the necessary dll's as specified in the twitchsdk README. Also, make sure to set the PATH environment variable to point to the directory containing the dll's.
[04:52:02] [Client thread/ERROR]: Couldn't initialize twitch stream
[04:52:02] [Sound Library Loader/INFO]: Starting up SoundSystem...
[04:52:02] [Thread-6/INFO]: Initializing LWJGL OpenAL
[04:52:02] [Thread-6/INFO]: (The LWJGL binding of OpenAL.  For more information, see http://www.lwjgl.org)
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)
[04:52:03] [Thread-6/INFO]: OpenAL initialized.
[04:52:03] [Sound Library Loader/INFO]: Sound engine started
[04:52:04] [Client thread/INFO]: [OptiFine] Loading mcpatcher/font/ascii.properties
[04:52:04] [Client thread/INFO]: [OptiFine] Loading mcpatcher/font/ascii_sga.properties
[04:52:08] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.client.ForgeHooksClient
[04:52:10] [Client thread/INFO]: [OptiFine] Multitexture: false
[04:52:10] [Client thread/INFO]: [OptiFine] ConnectedTextures: mcpatcher/ctm/glass/glass.properties
[04:52:10] [Client thread/INFO]: [OptiFine] Multipass connected textures: false
[04:52:10] [Client thread/INFO]: [OptiFine] Multipass connected textures: false
[04:52:10] [Client thread/INFO]: [OptiFine] BetterGrass: Parsing default configuration optifine/bettergrass.properties
[04:52:12] [Client thread/INFO]: Created: 1024x512 textures-atlas
[04:52:12] [Client thread/INFO]: [OptiFine] Animated sprites: 14
[04:52:13] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.client.model.ModelLoader
[04:52:13] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.client.ItemModelMesherForge
[04:52:14] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.fml.client.registry.RenderingRegistry
[04:52:14] [Client thread/INFO]: [LabyMod-Vanilla] Transforming class net.minecraftforge.client.event.RenderWorldLastEvent
[04:52:14] [Client thread/INFO]: [LabyMod-Vanilla] Transforming class net.minecraftforge.client.event.GuiScreenEvent$DrawScreenEvent$Post
[04:52:14] [Client thread/INFO]: [LabyMod-Vanilla] Transforming class net.minecraftforge.client.event.GuiScreenEvent$DrawScreenEvent
[04:52:14] [Client thread/INFO]: [LabyMod-Vanilla] Transforming class net.minecraftforge.client.event.GuiScreenEvent
[04:52:14] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.common.ForgeModContainer
[04:52:15] [Client thread/INFO]: [LabyMod-Vanilla] Transforming class net.minecraftforge.client.event.FOVUpdateEvent
EGLBridge: Created CTX pointer = 0x79c3eed600
[04:52:19] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.fml.common.FMLCommonHandler
[04:52:19] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.fml.common.asm.transformers.BlamingTransformer
[04:52:19] [Client thread/INFO]: [OptiFine] (Reflector) Class not present: net.minecraftforge.fml.relauncher.CoreModManager
[04:52:19] [Client thread/INFO]: ---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 11/8/21 4:52 AM
Description: Initializing game

java.lang.UnsatisfiedLinkError: Can't obtain static method fromNative(Method, Object) from class com.sun.jna.Native
	at com.sun.jna.Native.initIDs(Native Method)
	at com.sun.jna.Native.<clinit>(Native.java:135)
	at com.sun.jna.Structure.<clinit>(Structure.java:146)
	at net.labymod.discordapp.DiscordApp.<init>(DiscordApp.java:44)
	at net.labymod.main.LabyMod.init(LabyMod.java:306)
	at ave.am(SourceFile:470)
	at ave.a(SourceFile:310)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at com.sun.jna.Native.initIDs(Native Method)
	at com.sun.jna.Native.<clinit>(Native.java:135)
	at com.sun.jna.Structure.<clinit>(Structure.java:146)
	at net.labymod.discordapp.DiscordApp.<init>(DiscordApp.java:44)
	at net.labymod.main.LabyMod.init(LabyMod.java:306)
	at ave.am(SourceFile:470)

-- Initialization --
Details:
Stacktrace:
	at ave.a(SourceFile:310)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchw
rapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.8.9
	Operating System: Linux (aarch64) version Android-11
	Java Version: 1.8.0-internal, Oracle Corporation
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 343411136 bytes (327 MB) / 483917824 bytes (461 MB) up to 483917824 bytes (461 MB)
	JVM Flags: 3 total; -Xms512M -Xmx512M -Xbootclasspath/p:/storage/emulated/0/Android/data/net.kdt.pojavlaunch/files/caciocavallo/ResConfHack.jar:/storage/emulated/0/Android/data/net.kdt.pojavlaunch/files/caciocavallo/cacio-androidnw-1.10-SNAPSHOT.jar:/storage/emulated/0/Android/data/net.kdt.pojavlaunch/files/caciocavallo/cacio-shared-1.10-SNAPSHOT.jar
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: 1.8
	LWJGL: 3.2.3 SNAPSHOT
	OpenGL: GL4ES wrapper GL version 2.1 gl4es wrapper 1.1.5, ptitSeb
	GL Caps: VboRegions not supported, missing: OpenGL 1.3, ARB_copy_buffer
Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because ARB_framebuffer_object is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: !        Â§4Magma 32x Â§bV2.zip
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	CPU: 9x null
	OptiFine Version: OptiFine_1.8.9_HD_U_M5
	OptiFine Build: 20210124-163719
	Render Distance Chunks: 2
	Mipmaps: 0
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 2.1 gl4es wrapper 1.1.5
	OpenGlRenderer: GL4ES wrapper
	OpenGlVendor: ptitSeb
	CpuCount: 8
[04:52:19] [Client thread/INFO]: #@!@# Game crashed! Crash report saved to: #@!@# /storage/emulated/0/Android/data/net.kdt.pojavlaunch/files/.minecraft/crash-reports/crash-2021-11-08_04.52.19-client.txt
[04:52:19] [Client thread/ERROR]: Unable to launch
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0-internal]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0-internal]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0-internal]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0-internal]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.sun.jna.Native
	at java.lang.Class.forName0(Native Method) ~[?:1.8.0-internal]
	at java.lang.Class.forName(Class.java:264) ~[?:1.8.0-internal]
	at net.labymod.support.report.CrashReportHandler.getTargetAddonUUID(CrashReportHandler.java:87) ~[CrashReportHandler.class:?]
	at net.labymod.support.report.CrashReportHandler.report(CrashReportHandler.java:40) ~[CrashReportHandler.class:?]
	at BytecodeMethods.reportCrash(BytecodeMethods.java:559) ~[BytecodeMethods.class:?]
	at ave.c(SourceFile:628) ~[ave.class:?]
	at ave.a(SourceFile:314) ~[ave.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:124) ~[Main.class:?]
	... 6 more
[04:52:19] [Client Shutdown Thread/INFO]: Stopping server

**Comment by Mathias-Boulay** [comment]
Labymod 1.8.9 not supported, not our fault.
